subprojects/distributions/src/changes/changes-since-4.0.txt

```diff
@@ -0,0 +1,5 @@
+Implemented interface org.gradle.api.initialization.IncludedBuild;@Incubating interface has been removed
+Method org.gradle.api.initialization.IncludedBuild;@Incubating interface has been removed
+Class org.gradle.api.invocation.Gradle;All regressions accepted
```

Comment:
I would really like to specify which regressions have been accepted for a class. If we accept the whole class then new binary breaking changes could appear without us noticing.